apps/myjobs/app/models/workflow_helper.rb

```diff
@@ -0,0 +1,52 @@
+class WorkflowHelper
```

Comment:
I know I suggested this name, but I now realize that `WorkflowHelper` could be confused with `WorkflowHelpers`.

One refactor suggestion would be to turn this into a `WorkflowFile` object instead, whose single attribute is a path to the file.

Then the class might look something like this:

```ruby
class WorkflowFile
  attr_reader :path

  def initialize(path)
    @path = path
  end

  def valid_script?
    @valid_script ||= valid_size?
  end

  def suggested_script?
  end

  def valid_size?
    File.size(path).to_f/1024 <= 65
  end
```

Then on `Workflow` you could add:

```ruby
def files
  if File.directory?(staged_dir)
    Find.find(staged_dir).drop(1).select {|f| File.file?(f) }.map {|f| WorkflowFile.new(f) }
  else
    []
  end
end
```

Reply:
Or don't do `.select {|f| File.file?(f) }` and then do that in WorkflowFile (with the logic you have now - if it isn't a File it can't be a valid script.

Reply:
The new commit fixes this feedback by adding a new WorkflowFile model and removing the WorkflowHelper